frameworks/digital-outcomes-and-specialists/questions/briefs/workplace.yml

```diff
@@ -1,6 +1,6 @@
-name: Important dates
-question: Any important dates
-hint: 'eg a policy date by which the minimum viable product must be ready'
+name: Address where the work will take place
```

Comment:
Question name should probably reference the fact that it's supposed to be an address